### ISSUE TITLE
Rename topic SIGHASH_NOINPUT to SIGHASH_ANYPREVOUT

### ISSUE THREAD

**Issue by harding** [bitcoinops/bitcoinops.github.io#605]
We haven't done a topic rename before but I think it's something we're going to need to do occasionally.  Things to consider:

1. Need to create a redirect link from the old URL to the new URL
2. Need to mention the old name on the new page so people going to the old URL don't get too confused
3. Need to either not break our automatic markdown reference links or update them all

**Comment by jnewbery** [comment]
> 3. Need to either not break our automatic markdown reference links or update them all

Renaming them all in the same commit doesn't seem too difficult. Presumably a `make clean && make` would catch any that we miss?

**Comment by harding** [comment]
@jnewbery indeed.  Even easier I think would be just to add `[topic sighash_noinput]: /en/topics/sighash_anyprevout/` to `_includes/references.md`.  The tests would also catch any place where that didn't work.
